tools/figma-inspector/src/main.tsx

```diff
@@ -102,40 +102,63 @@ export const App = () => {
                     />
                 </DialogFrame.Content>
                 <DialogFrame.Footer>
-                    <div className="border border-secondary  border-[var(--figma-color-bg-brand)]">
-                        <Text className="text-[var(--figma-color-bg-brand)]">
-                            Welcome to Figma
-                        </Text>
-                    </div>
+                    <Checkbox.Root>
+                        <Checkbox.Input
+                            checked={useVariables}
+                            onChange={(e) => setUseVariables(e.target.checked)}
+                        />
+                        <Checkbox.Label>Use Figma Variables</Checkbox.Label>
+                    </Checkbox.Root>
+
```

Comment:
This seems unrelated. Did you mean to include the changes in this file in the commit, or did you intend to make a separate change?

Reply:
Ah, I See, this PR is on top of https://github.com/slint-ui/slint/pull/8321 . I suggest to either get that in first or rebase this onto master without the changes in #8321 .